docs/source/getting_started.rst

```diff
@@ -0,0 +1,171 @@
+Getting started
+===============
+
+For installation of the Ankaios SDK, see the `Installation section <index.html#installation>`_.
+
+Once the SDK is installed, you can start using it by importing the module and creating an ankaios object.
+
+.. code-block:: python
+
+    from ankaios import Ankaios
+
+    ankaios = Ankaios()
+
+The initialization of the Ankaios object will automatically connect to the fifo pipes of the control interface. Once this is done,
+the communication with the Ankaios ecosystem can be started.
+
+**Apply a manifest**
+--------------------
+
+Considering we have a manifest file with a workload called ``nginx_test`` and a config called ``test_ports``. The manifest file is as follows:
+
+.. code-block:: yaml
+    :caption: my_manifest.yaml
+
+    apiVersion: v0.1
+    workloads:
+        nginx_test:
+            runtime: podman
+            restartPolicy: NEVER
+            agent: agent_A
+            configs:
+                ports: test_ports
+            runtimeConfig: |
+                image: image/test
+    configs:
+        test_ports:
+            port: \"8081\"
+
+The manifest can now be applied using the following code:
+
+.. code-block:: python
+
+    from ankaios import Ankaios, Manifest
+
+    # Create an Ankaios object
+    ankaios = Ankaios()
+
+    # Load the manifest from the file
+    manifest = Manifest.from_file('my_manifest.yaml')
+
+    # Apply the manifest and get the result
+    ret = ankaios.apply_manifest(manifest)
+
+    # Get the workload instance name
+    wl_instance_name = ret["added_workloads"][0]
+
+    # Print the instance name
+    print(wl_instance_name)
+
+If the operation is succesfull, the result will contain a list with the added workloads that contains the workload instance name of our own.
+The workload instance name contains the name of the workload, the agent it is running on and an unique identifier.
+
+**Update a workload**
+---------------------
+
+Considering we have the above workload running, we can update certain parameters of the workload. For this example, we will update the `restartPolicy`. To be able to pinpoint
+the exact workload we want to modify, we must know the workload instance name. This can be obtained as a result when starting the workload (either using `apply_manifest` or other methods),
+deleting or modifying one. In case we don't have the workload instance name, we can take all the workloads that have the same name as the one we are looking for (or the agent).
+For simplicity, we will consider that the workload instance name is known.
```

Comment:
Let's keep it simple: ```suggestion
the exact workload we want to modify, we must know the workload instance name. This can be obtained as a result when starting the workload, deleting or modifying one.
For simplicity, we will consider that the workload instance name is known.
```